db/migrate/20180723091143_add_phone_confirmed_to_otp_requests_tracker.rb

```diff
@@ -0,0 +1,16 @@
+class AddPhoneConfirmedToOtpRequestsTracker < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def up
+    add_column :otp_requests_trackers, :phone_confirmed, :boolean
+    change_column_default :otp_requests_trackers, :phone_confirmed, false
```

Comment:
This works, but why not just put this on the previous line with `default: false`?

Reply:
Strong migrations prevented it.